packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -36,6 +36,7 @@ import './PipelineRuns.scss';
 const PipelineRuns = ({
   cancelPipelineRun,
   createPipelineRunURL = urls.pipelineRuns.byName,
+  createPipelineRunName = ({ pipelineRunMetadata }) => pipelineRunMetadata.name,
```

Comment:
maybe `createPipelineRunDisplayName` since we're not actually modifying the name of the run?